test/md055-missing-pipes.md

```diff
@@ -0,0 +1,19 @@
+# MD055 Missing Borders
```

Comment:
Please add tests for missing pipe on the header line and the separator line, too.

Reply:
ok, I'll try, but as mentioned already some of that won't be recognized as a table anymore.